app/models/post.rb

```diff
@@ -0,0 +1,19 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_count
+
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
+  def update_posts_count
+    author.increment!(:posts_counter)
+  end
```

Comment:
- [ optional ] Excellent job using the callback **after_save** on the `update_posts_count` method. Ill just recommend setting it as private as you did with the `update_like_count` in like.rb